samples/tab-deeplink-all-features/nodejs/appPackage/manifest.json

```diff
@@ -0,0 +1,51 @@
+{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.12/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.12",
+  "version": "1.0",
+  "id": "<<Microsoft-App-ID>>",
+  "packageName": "com.microsoft.deeplink",
+  "developer": {
+    "name": "Microsoft",
+    "websiteUrl": "https://www.microsoft.com",
+    "privacyUrl": "https://www.microsoft.com/privacy",
+    "termsOfUseUrl": "https://www.microsoft.com/termsofuse"
+  },
+  "name": {
+    "short": "Deeplink All Features",
+    "full": "All features for Deeplink using tab"
+  },
+  "description": {
+    "short": "DeepLink with all features for Microsoft Teams",
+    "full": "This sample DeepLink app for demonstrating deeplink all features using tab."
```

Comment:
"short": "Deep Links with different features for Microsoft Teams",
    "full": "This sample app demonstrates deep links features using tab."

Reply:
fixed